now/executor/gateway/bff/app/v1/models/search.py

```diff
@@ -26,10 +26,10 @@ class SearchRequestModel(BaseRequestModel):
     limit: int = Field(
         default=10, description='Number of matching results to return', example=10
     )
-    filters: Optional[Dict[str, str]] = Field(
+    filters: Optional[Dict[str, Union[List, Dict[str, int]]]] = Field(
```

Comment:
```suggestion
    filters: Optional[Dict[str, Union[List, Dict[str, float]]]] = Field(
```
since in the example provided value is float type?

Reply:
It could be float or int actually! I can fix that.